ci/jenkins/utils.sh

```diff
@@ -14,6 +14,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+function check_and_cleanup_buildkit_cache() {
+    free_space=$(df -h -B 1G / | awk 'NR==2 {print $4}')
+    free_space_threshold=40
+    if [[ $free_space -lt $free_space_threshold ]]; then
+        # If cleaning up unused dangling images doesn't free up sufficient disk space,
+        # we will have to reduce the buildkit cache to 10GB to release enough disk space.
+        docker buildx prune -af --keep-storage=10gb > /dev/null
```

Comment:
so all Jenkins workers can already run `docker buildx`?

Reply:
No it doesn't work with some legacy docker versions, the command failure won't block the CI pipeline. However, given that docker builder prune can achieve the same function and can be supported on all docker releases, I have updated it for better compatibility.